lib/tasks/sidekiq.rake

```diff
@@ -43,14 +43,28 @@ namespace :sidekiq do
         dest: RedisJobSet.new(redis: dest_redis),
         logger: logger,
         progress: progress,
-        )
+      )
 
       delayed_queue = SidekiqMover.new(
         source: SidekiqDelayedQueue.new(redis: source_redis),
         dest:   SidekiqDelayedQueue.new(redis: dest_redis),
         logger: logger,
         progress: progress
-        )
+      )
+
+      retry_queue = SidekiqMover.new(
+        source: SidekiqDelayedQueue.new(key: 'retry', redis: source_redis),
+        dest: SidekiqDelayedQueue.new(key: 'retry', redis: dest_redis),
```

Comment:
Minor niggle: indentation of `dest:` arg here and below doesn't match other calls.